types/mathjs/index.d.ts

```diff
@@ -537,18 +537,25 @@ declare namespace math {
         /**
          * Simplify an expression tree.
          * @param expr The expression to be simplified
-         * @param rules A list of rules are applied to an expression, repeating
+         * @param rules (optional) A list of rules are applied to an expression, repeating
          * over the list until no further changes are made. It’s possible to
          * pass a custom set of rules to the function as second argument. A rule
          * can be specified as an object, string, or function.
-         * @param scope Scope to variables
+         * @param scope (optional) Scope to variables
+         * @param options (optional) Simplify options object with two knobs:
+         *   - `exactFractions`: a boolean which is `true` by default.
+         *   - `fractionsLimit`: when `exactFractions` is true, a fraction will be returned
```

Comment:
Move these lines to jsdoc of fields of `SimplifyOptions` should be better

Reply:
I see your point and changed accordingly.